tests/011-test-docker-root-volume-cleanup.sh

```diff
@@ -21,6 +21,7 @@ test_reset_docker_root_volume() {
 DEVS="$devs"
 VG=$vg_name
 DOCKER_ROOT_VOLUME=yes
+DOCKER_ROOT_VOLUME_SIZE=20%FREE
```

Comment:
Any specific reason we are setting this to `20%FREE`? The default value should automatically get picked.